src/lib/migration/proposal_store.rb

```diff
@@ -0,0 +1,230 @@
+    def handle_link(link)
+      client = @id_mapping[link]
+      client ||= @link2submod[link]
+
+      if !client
+        log.error "unknown link #{link}. Broken proposal client?"
+        return nil
+      end
+
+      data = {
+        "has_next"  => false,
+        "chosen_id" => link
+      }
+
+      Yast::WFM.CallFunction(client, ["AskUser", data])
+    end
+
+  private
+
+    def global_help
+      case @proposal_mode
+      when "initial"
+        if Yast::Mode.installation
+          # Help text for installation proposal
+          # General part ("You can change values...") is added as the next paragraph.
+          _(
+            "<p>\n" \
+              "Select <b>Install</b> to perform a new installation with the values displayed.\n" \
+              "</p>\n"
+            )
+        else # so update
+          # Help text for update proposal
+          # General part ("You can change values...") is added as the next paragraph.
+          _(
+            "<p>\n" \
+              "Select <b>Update</b> to perform an update with the values displayed.\n" \
+              "</p>\n"
+          )
+        end
+      when "network"
+        # Help text for network configuration proposal
+        # General part ("You can change values...") is added as the next paragraph.
+        _(
+          "<p>\n" \
+            "Put the network settings into effect by pressing <b>Next</b>.\n" \
+            "</p>\n"
+        )
+      when "service"
+        # Help text for service configuration proposal
+        # General part ("You can change values...") is added as the next paragraph.
+        _(
+          "<p>\n" \
+            "Put the service settings into effect by pressing <b>Next</b>.\n" \
+            "</p>\n"
+        )
+      when "hardware"
+        # Help text for hardware configuration proposal
+        # General part ("You can change values...") is added as the next paragraph.
+        _(
+          "<p>\n" \
+            "Put the hardware settings into effect by pressing <b>Next</b>.\n" \
+            "</p>\n"
+        )
+      when "uml"
+        # Proposal in uml module
+        _("<P><B>UML Installation Proposal</B></P>") \
+        # help text
+        _(
+          "<P>UML (User Mode Linux) installation allows you to start independent\nLinux virtual machines in the host system.</P>"
+        )
+      else
+        if properties["help"] && !properties["help"].empty?
+          # Proposal help from control file module
+          Yast::Builtins.dgettext(
+            @proposal_text_domain
+            properties["help"]
+          )
+        else
+          # Generic help text for other proposals (not basic installation or
+          # hardhware configuration.
+          # General part ("You can change values...") is added as the next paragraph.
+          _(
+            "<p>\n" \
+              "To use the settings as displayed, press <b>Next</b>.\n" \
+              "</p>\n"
+          )
+        end
+      end
```

Comment:
no, it is specific helps for different proposal modes ( one mode is installation one, others are usually deprecated ). Control file used to have more modes defined and proposal_mode switched it.